src/Core/Settings.h

```diff
@@ -872,6 +872,7 @@ class IColumn;
     M(UInt64, cache_warmer_threads, 4, "Only available in ClickHouse Cloud", 0) \
     M(Int64, ignore_cold_parts_seconds, 0, "Only available in ClickHouse Cloud", 0) \
     M(Int64, prefer_warmed_unmerged_parts_seconds, 0, "Only available in ClickHouse Cloud", 0) \
+    M(Bool, enable_order_by_all, true, "Enable sorting expression ORDER BY ALL.", 0) \
```

Comment:
Is this setting in the experimental group by mistake or is it really experimental?

Reply:
By mistake (though a few settings in the section don't look experimental).

Fix: https://github.com/ClickHouse/ClickHouse/pull/60449